eclair-core/src/main/scala/fr/acinq/eclair/channel/fund/InteractiveTxBuilder.scala

```diff
@@ -114,6 +115,31 @@ object InteractiveTxBuilder {
     val minNextFeerate: FeeratePerKw = targetFeerate * 25 / 24
   }
 
+  // @formatter:off
+  sealed trait Purpose {
+    def previousLocalBalance: MilliSatoshi = 0.msat
+    def previousRemoteBalance: MilliSatoshi = 0.msat
+    def commitmentIndex: Long = 0
+    def perCommitmentPoint: PublicKey
```

Comment:
Is this local or remote? I believe it's remote, it's worth being explicit with the name, isn't it?
```suggestion
    def remotePerCommitmentPoint: PublicKey
```

Reply:
Renamed in d47aec354.